tests/ui/diagnostic_namespace/on_unimplemented/ignore_unsupported_options_and_continue_to_use_fallback.stderr

```diff
@@ -22,8 +23,9 @@ LL | |     if(Self = ()),
 ...  |
 LL | |     note = "not used yet"
 LL | | )]
-   | |__^
+   | |__^ invalid option found here
```

Comment:
Should be able to point at the `if(Self = ())` here.

Reply:
I've pushed a "fix" to address this somewhat. The problem there is that `if(Self = ())` doesn't parse as `meta_item_list()`. I'm not sure why that's the case or whether that's the right behavior. The pushed "fix" just turns the attribute into something that can be parsed (`if(Self = "()")` in that case).

Reply:
oh, ok